mbedtls-sys/Cargo.toml

```
@@ -26,6 +26,9 @@ libc = { version = "0.2.0", optional = true }
 [target.'cfg(unix)'.dependencies]
 libc = { version = "0.2.0" }
 
+[target.'cfg(windows)'.dependencies]
+libc = { version = "0.2.0" }
```

Just a small suggestion, you could consider merging the following two cfg blocks into one:
```suggestion
[target.'cfg(any(unix, windows))'.dependencies]
libc = { version = "0.2.0" }
```